### PR TITLE
Update monitor-connectivity.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Microsoft System Center Operations Manager Management Pack for Microsoft 365

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6138

File: microsoft-365/enterprise/monitor-connectivity.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Once you've deployed Microsoft 365, you can maintain Microsoft 365 connectivity
...
-|**Use System Center to Monitor Microsoft 365** <br/> |If you're using Microsoft System Center, you can download the [System Center Management Pack for Office 365](https://www.microsoft.com/download/details.aspx?id=43708) to begin monitoring Microsoft 365 today. For more detailed guidance, please see the management pack operations guide. <br/> |
+|**Use System Center to Monitor Microsoft 365** <br/> |If you're using Microsoft System Center, you can download the [Microsoft System Center Operations Manager Management Pack for Microsoft 365](hhttps://www.microsoft.com/en-us/download/details.aspx?id=103379) to begin monitoring Microsoft 365 today. For more detailed guidance, please see the management pack operations guide. <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/monitor-connectivity.md]:
```suggestion
|**Use System Center to Monitor Microsoft 365** <br/> |If you're using Microsoft System Center, you can download the [Microsoft System Center Operations Manager Management Pack for Microsoft 365](https://www.microsoft.com/download/details.aspx?id=103379) to begin monitoring Microsoft 365 today. For more detailed guidance, please see the management pack operations guide. <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**